notebooks/ch-paper-implementations/tsp.ipynb

```diff
@@ -933,9 +933,9 @@
     "0 & 0 & 0 & 0 & 0 & 0 & 0 & e^{id}\n",
     "\\end{bmatrix}\n",
     "$$\n",
-    "but kind of rearranged version of this. Though the matrix we are getting and the $CU_{j}$ matrix is mathematically same. This is because Qiskit interpret the tensor product from bottom qubit to top qubit. So to get exactly same form as the $CU_{j}$, we have to make the circuit inverted.\n",
+    "but a rearranged version of this. However, the matrix we get and the $CU_{j}$ matrix are mathematically identitcal. This is because Qiskit interprets the tensor product from bottom qubit to top qubit. So to get the exact same form as the $CU_{j}$, we have to make the circuit inverted.\n",
```

Comment:
```suggestion
    "but a rearranged version of this. However, the matrix we get and the $CU_{j}$ matrix are mathematically identical. This is because Qiskit interprets the tensor product from bottom qubit to top qubit. So to get the exact same form as the $CU_{j}$, we have to make the circuit inverted.\n",
```